maps/utk.yml

```diff
@@ -2363,6 +2344,8 @@ properties:
     property_uri: http://purl.org/dc/terms/language
     range: http://www.w3.org/2001/XMLSchema#anyURI
     requirement: optional
+    sample_values:
+    - English
```

Comment:
Range here is URI but suggest value is string.

Decision in MODS to RDF says it should be a URI:
https://utk-mods-to-rdf.readthedocs.io/en/latest/contents/4_mapping.html#id221

Reply:
Wanted to confirm here - I think we do want URIs as the value, but currently our metadata only includes strings. Is it okay to assume that UTK (me or you) will change all string values to URIs during migration/ingest? Attached are the existing values. I've updated the example to URI so this is no longer in conflict.
[language.txt](https://github.com/utkdigitalinitiatives/m3_profiles/files/9982548/language.txt)

Reply:
@mlhale7, my migration utilities do convert to URI, but they do not currently write a value if one does not exist.  If it should do that, I think we need to have explicit instructions on what it should do per work type class.

Reply:
Perfect - I didn't realize this. I don't think it's possible to accurately write one in if no value currently exists. It does no look like this M3 profile took in the changes I made to the spreadsheet on obligation as language still has a minimum requirement of 1. I may need to open another PR and review obligation to make sure everything is accurate.